docs/source/conf.py

```diff
@@ -211,7 +212,7 @@
     "link_voxel51_slack": "https://slack.voxel51.com",
     "link_voxel51_twitter": "https://twitter.com/voxel51",
     "link_voxel51_blog": "https://voxel51.com/blog/",
-    "banner_link": "https://docs.voxel51.com/integrations/coco.html",
+    "og_image": "https://voxel51.com/wp-content/uploads/2024/02/1.24_Webpages_MeetupLandingSC_AV.png",
```

Comment:
The meetup image isn't appropriate for docs pages. How about the home page image? lets confirm on slack
```suggestion
    "og_image": "https://voxel51.com/wp-content/uploads/2024/03/3.24_webpages_Home_AV.png",
```